PrestaShop/PrestaShop
# Simplify stock_threshold_alert field in UpdateProductCommand and UpdateCombinationCommand

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | https://github.com/PrestaShop/PrestaShop/issues/30750
| Type?             | refacto
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | See bellow
| Fixed ticket?     | Closes https://github.com/PrestaShop/PrestaShop/issues/30750
| Related PRs       | If theme, autoupgrade or other module change is needed to make this change work, provide a link to related PRs here.
| Sponsor company   | Your company or customer's name goes here (if applicable).

**How to test**
In product page v2 edit product stock tab - make sure that low stock alert switch and low stock level input works as intended (when alert switch is on then you can input the low stock level value, else you cannot and the threshold is not updated)
Same with combination form and combination bulk edition form.
